modules/decision_reviews/app/sidekiq/decision_reviews/failure_notification_email_job.rb

```diff
@@ -51,29 +45,38 @@ def perform_form_and_evidence
 
     def perform_all
       enabled? &&
-        secondary_forms_enabled? &&
         (submissions.present? || submission_uploads.present? || errored_secondary_forms.present?)
     end
 
-    def vanotify_service
-      @service ||= ::VaNotify::Service.new(VANOTIFY_API_KEY)
+    def get_callback_config(email_type, appeal_type)
+      case email_type
+      when :form
+        [DecisionReviews::FormNotificationCallback, 'form submission', DecisionReviews::V1::FORM_TEMPLATE_IDS[appeal_type]]
+      when :evidence
+        [DecisionReviews::EvidenceNotificationCallback, 'evidence submission to Lighthouse', DecisionReviews::V1::EVIDENCE_TEMPLATE_IDS[appeal_type]]
+      when :secondary_form
+        [DecisionReviews::EvidenceNotificationCallback, 'secondary form submission to Lighthouse', DecisionReviews::V1::SECONDARY_FORM_TEMPLATE_ID]
+      end
     end
 
-    def vanotify_service_with_callback(submission, template_id)
+    def vanotify_service_with_callback(submission, email_type, reference)
+      appeal_type = submission.type_of_appeal
+      callback_klass, function, email_template_id = get_callback_config(email_type, appeal_type)
+      service_name = DecisionReviews::V1::APPEAL_TYPE_TO_SERVICE_MAP[appeal_type]
+
       callback_options = {
-        callback_klass: DecisionReviews::FormNotificationCallback.to_s,
+        callback_klass: callback_klass.to_s,
         callback_metadata: {
           email_type: :error,
-          service_name: APPEAL_TYPE_TO_SERVICE_MAP[submission.type_of_appeal],
-          function: 'form submission',
+          service_name:,
+          function:,
           submitted_appeal_uuid: submission.submitted_appeal_uuid,
-          email_template_id: template_id
+          email_template_id:,
+          reference:,
+          statsd_tags: ["service:#{service_name}", "function:#{function}"]
```

Comment:
This addition should address the errors we were seeing that were being triggered from the [VANotify::DefaultCallback class' validate_and_normalize_statsd_tags ](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/va_notify/lib/va_notify/default_callback.rb#L89)method:

> VANotify: Invalid metadata format: Invalid metadata statsd_tags format: must be a Hash or Array

While we only rely on the VANotify::CustomCallback functionality to execute our expected callback behavior, when the VA Notify callbacks endpoint is triggered, [it initializes a VANotify::DefaultCallback](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/va_notify/app/controllers/va_notify/callbacks_controller.rb#L30) and triggers the call method on it as well.